### DISCUSSION TITLE
My account got suspended without any notes

### DISCUSSION THREAD

**Discussion by ghost** [community/community#49835] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Yesterday,My Github account: https://github.com/SayedAlAhlMohamed
was suspended.
I don't know why as I didn't receive any mails or hints indicating why this happened.
I was taking to some one here in the community trying to solve a problem when suddenly it logged me out and when tried to sign in a message appear that account suspended for violating GitHub’s terms of service.Can some one please what is happening?
I am so sad and can't describe my sadness. I never violated TOS in my life and if that happened it is absolutely a mistake and not intentional. I’m happy to oblige with all of GitHub’s policies and Terms.

I raised A ticket to support with this ID: Ticket ID: 2049915

Can some one from support Help me Recover the account and Tell me why my account got suspended?
![024](https://user-images.githubusercontent.com/127531635/224517962-c066ae47-ecb2-452b-be3a-156c663379b1.png)

**Comment by ghost** [comment]
@queenofcorgis 

please, Can you help me solve the issue?

**Comment by Abinashbunty** [comment]
I think it's because you used some sort of AI to provide generalised answers to the questions in the Discussions. This might have been violation of conditions. 

You can refrain from answering and that is completely fine of course. But providing multiple answers to various questions on Discussions which don't nearly solve the problem is quite wrong. Especially considering the fact that same grammar and same approach is used in all your answers and those don't involve solving the question. Rather they provide random solution which of course the author has checked from the internet.

Moreover again [here](https://github.com/orgs/community/discussions/49818), you're marking your own reply as an answer. That's again another violation.

**Reply by ghost** [reply]
I didn't know that using AI is forbidden and I made sure that the info is true most of the time. I even managed to solve more than 23 Question here so I am not spamming or wasting anyone time.
Maybe what i did is wrong and I am happy to oblige with all of GitHub’s policies and Terms, but they didn't give me a warning or message indicating that is wrong. Many people marked my answer as the answer and they were very glad and I was happy to help the commmunity and make friends as i don't have any.

![photo_2023-03-10_18-32-01](https://user-images.githubusercontent.com/127531635/224518674-bf25febc-fdb2-436a-97f8-9c3d17db2066.jpg)
![photo_2023-03-12_03-05-18](https://user-images.githubusercontent.com/127531635/224518675-e2808a6d-4427-46db-bb1f-c00a788c5fa9.jpg)

**Reply by Abinashbunty** [reply]
That's super cool if people were able to get answers to their questions. 😀 I'm not sure if using AI is forbidden. After all, Github also has Copilot. But I think some users who asked questions found your answers as spam maybe and that resulted in account getting suspended. 

Wait for reply from GitHub staff. 👍🏻 Hopefully it'll be solved soon.

**Reply by ghost** [reply]
@Abinashbunty 
Thanks bro. 
I searched yesterday to make sure but didn't find anything saying that is forbidden ,but I am waiting for Github staff to answer an tell me and if it is forbidden I am happy to comply to all of GitHub’s policies and Terms.
I hope they solve it soon. I really loved that account and am really sad to see it gone like this

**Comment by ghost** [comment]
@github-staff 
@github
Can Anyone Help me ,please?

**Comment by ghost** [comment]
It appears to be related to having more than one account. When support just didn't answer me two months ago. I made this account and it appears that is against the TOS. I didn't know about that and I am not justifying my reason as it like I said taked two months and by chance opened an old mail now to find out it just got a respond at the end of  the last month.
I want to let you know guys I am ready to comply with all Github Rules and Terms and Conditions and It wasn't my intention.
I want to delete all my accounts and recover this one only : https://github.com/Sayedalahlmohamed
can you do this for me? 
@github-staff
@github
@queenofcorgis 
@Akash1134

I hope you guys understand my situation and help me. I am always happy to oblige to the rules 

Thx in advance!!

**Reply by 0x73746F66** [reply]
Literally every client I worked with in the past would be violating this rule for most of their GitHub users!

Every GitHub user is technically a personal account, that are essentially associated to one or more organisations, so many corporate run organisations ask you to create a GitHub user (which is always a personal user, there's no such thing as a corporate owned user) using the corporate controlled email address.

So I and many others have many duplicate GitHub users because that is the way most, if not all, corporate organisations use the product.

That rule is like Facebook saying ti is a violation to use the "post something" feature, everyone who uses it because that's how you use the product, would unknowingly be violating the rules... it's stupid, and GitHub wouldn't have a stupid rule that normal users would be violating through normal use would they?

**Reply by ghost** [reply]
@0x73746F66
I received mail from support lately on another account that indicates it is against TOS.and they just archived the ticket,so I couldn't understand anything from them.
This may or may Not be the reason for my suspension ,but I am just speculating as no one is answering me.

**Reply by ghost** [reply]
And Despite me wanting to oblige to the rules and delete all my accounts , they still didn't answer!!
Like they are saying hey violate it again we won't answer!!

**Comment by ghost** [comment]
As they all related and I am just asked to delete all my accounts. IF that requires a ticket I have opened a ticket long time ago and It appear to be lost or something as I didn't get any respond for almost 2 months now. So i added my request to it to delete all accounts and leave only this one: https://github.com/Sayedalahlmohamed
Can anyone just escalate this issue of delay in response  so that i can have my account back and comply to all TOS. And is it okay to add something to the ticket?

Ticket ID: #1977213

**Comment by ghost** [comment]
Any help or suggestion to solve the problem will be really appreciated?!
@github-staff

**Comment by ghost** [comment]
up

**Comment by Waltonsteven07** [comment]
Hello!! I got this friend from high school, Chloe William's, she handles meta account recovery and works with meta ic3 help team, she’s a full time computer technician at meta and persons with account problems can look her up on Instagram or telegram @chloe_consultant to help you regain access to your account that was hacked or disabled. you can also send her Email; chloe.consultant1@gmail.com You got other cyber related problems or need a PI? Reach out to her for assistance.

**Comment by queenofcorgis** [comment, marked as answer]
The community is unable to assist with nor escalate your tickets, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to lock this discussion.

Thank you!
